### ISSUE TITLE
Uncaught ERROR: multiple element <head> occourences are not supported (found 2 occourences)

### ISSUE THREAD

**Issue by adixon** [veda-consulting/uk.co.vedaconsulting.mosaico#624]
1. Drupal 10.1.6 + CiviCRM 5.65.0
2. Drupal is multilingual with path prefix
3. Mosiaco upgraded to 3.3.1697392242

New mailings using versafix-1 are okay.
New mailings using one of the other custom templates based on versafix give the error in the console:

`Uncaught ERROR: multiple element <head> occourences are not supported (found 2 occourences)`

Found in the file: /sites/default/files/civicrm/ext/uk.co.vedaconsulting.mosaico/packages/mosaico/dist/rs/src/js/template-loader.js

I can create new templates from versafix, but when trying to edit them and any other custom template based on versafix, I get the same error above.

I reviewed the entry in civicrm_mosaico_template and the html doesn't have multiple <head> values in the html, it has exactly one.

So I suspect the js error is misleading, and I'm going to hazard a guess that it's related to the path prefix thingy (but only due to recency bias on my part).

We did disable a few mosaico extensions (tweaks and message templates), the error continues.

**Comment by adixon** [comment]
Perhaps this is the same issues as this one: https://github.com/veda-consulting-company/uk.co.vedaconsulting.mosaico/issues/622

**Comment by KarinG** [comment]
Screenshot -> when selecting an existing template:

<img width="1678" alt="image" src="https://github.com/veda-consulting-company/uk.co.vedaconsulting.mosaico/assets/5340555/c5c9046d-38d0-4ee4-98c9-0e98a69783fb">

**Comment by KarinG** [comment]
I don't believe there is a file named template-loader.js in 3.3 (neither on this client side; nor on a freshly downloaded 3.3 from https://civicrm.org/extensions/email-template-builder)

I can find other files:
karins-MBP:uk.co.vedaconsulting.mosaico sysadmin$ find . -name mosaico.min.js
./packages/mosaico/dist/rs/mosaico.min.js

Not this one:
karins-MBP:uk.co.vedaconsulting.mosaico sysadmin$ find . -name template-loader.js
karins-MBP:uk.co.vedaconsulting.mosaico sysadmin$

**Comment by adixon** [comment]
Yes, see the related issue I linked above, in that case they were resolved it by trying a different browser!

Trying to access the file via https similarly fails (i.e., not a server caching issue).

**Comment by KarinG** [comment]
Using a completely different browser does not make any difference in this case -> Safari:

<img width="1417" alt="image" src="https://github.com/veda-consulting-company/uk.co.vedaconsulting.mosaico/assets/5340555/09fb0804-779b-4c39-9088-e007c808d94b">

template-loader.js is not a file on a freshly installed version of Mosaico on my local instance either.

Checking a (third) completely different project that has a functioning Mosaico: that template-loader.js file does not exist either. 
root@5cf31e9a5401:/var/www/drupal/web/sites/default/files/civicrm/ext/uk.co.vedaconsulting.mosaico# find . -name template-loader.js
root@5cf31e9a5401:/var/www/drupal/web/sites/default/files/civicrm/ext/uk.co.vedaconsulting.mosaico# 

Why are we looking for this file? :-)

**Comment by adixon** [comment]
Yes, that's the question. Or rather, how the heck is a non-existent file not only being referenced, but also loaded (as seen in the javascript console) and generating errors?

The referenced issue suggested that it might be related to browser caching after an upgrade, but then again maybe not (as per your post).

In any case, presumably this is all connected to the fact that the extension is now using an updated version of the upstream mosaico project.

Conclusion: WTF?!

**Comment by demeritcowboy** [comment]
The template-loader.js seems to be bundled and minified into mosaico-libs-and-tinymce.min.js, and it may or may not be adding a problem, but there is a separate problem before that. I'll post on #567 since that seems the closest.
